p2pserver/link/link.go

```diff
@@ -166,20 +165,12 @@ func (this *Link) CloseConn() {
 	}
 }
 
-func (this *Link) Tx(msg types.Message) error {
+func (this *Link) Tx(msgType string, payload []byte) error {
```

Comment:
unused `msgType` .  the original method is convenient to call. rename it to `Send` and add another method ?
```golang
Send(msg types.Message) error // for common usage
SendRaw(rawPacket  []byte) error // for broadcast, avoid serialization many times 

```